packages/esm-ohri-pmtct-app/src/dashboard.meta.tsx

```diff
@@ -33,3 +35,24 @@ export const mchFolderMeta = {
   slotName: 'mch-slot',
   isExpanded: false,
 };
+
+// Clinical Dashboards
+// export const motherChildHealthFolderDashboardMeta = {
+//   slot: 'ohri-mother-child-health-dashboard-slot',
+//   config: { columns: 1, type: 'grid', icon: ParentChild },
+//   isFolder: true,
+//   title: 'Mother Child Health',
+// };
+
+export const motherChildDashboardMeta = {
+  name: 'mother-child-health',
+  slot: 'mother-child-health-dashboard-slot',
+  config: {
+    columns: 1,
+    type: 'grid',
+    programme: 'pmtct',
+    icon: ParentChild,
```

Comment:
This icon doesn't exist. Look for an alternative within the carbon library. This is the one that comes closest <PedestrianChild /> 

https://carbondesignsystem.com/guidelines/icons/library

Reply:
@pirupius It exists but has been removed since we aren't using a folder yet for the Maternal Child Health Clinical Dashboard